core/src/main/java/org/mule/runtime/core/internal/policy/CompositeOperationPolicy.java

```diff
@@ -130,4 +137,11 @@ public CoreEvent process(CoreEvent event) throws MuleException {
       return error(e);
     }
   }
+
+  private void logOperationEvent(EventContext eventContext, Message message) {
+    if (LOGGER.isTraceEnabled()) {
+      LOGGER.trace("Executing operation\n" + eventContext.toString() + "\n"
+          + message.getAttributes().getValue().toString());
```

Comment:
is the attributes `toString` worth logging? it is not implemented to show any meaningful data. Maybe the component path of the operation being executed provided more meaningful info.